# Use the LDFLAGS passed at configure time; needed to get MacPorts -arch flags.
--- configure.in.orig	2011-05-18 02:59:28.000000000 -0500
+++ configure.in	2011-08-17 05:12:47.000000000 -0500
@@ -707,6 +707,7 @@
 AC_SUBST(ASSEMBLY_SOURCES)
 AC_SUBST(CC)
 AC_SUBST(CFLAGS)
+AC_SUBST(LDFLAGS)
 AC_SUBST(SPECIFIC_LDFLAGS)
 AC_SUBST(SPECIFIC_CFLAGS)
 AC_SUBST(DCT_IA64_SOURCES)
--- platform.inc.in.orig	2011-01-11 05:39:13.000000000 -0600
+++ platform.inc.in	2011-08-17 05:10:02.000000000 -0500
@@ -33,6 +33,7 @@
 SPECIFIC_CFLAGS=@SPECIFIC_CFLAGS@
 ALTIVEC_CFLAGS=@ALTIVEC_CFLAGS@
 CFLAGS=@CFLAGS@
+LDFLAGS=@LDFLAGS@
 
 # ============================================================================
 # Assembler
